core/uimarket/src/main/java/ir/composenews/uimarket/model/MarketModel.kt

```diff
@@ -1,5 +1,8 @@
 package ir.composenews.uimarket.model
 
+import android.os.Parcel
+import android.os.Parcelable
+
 data class MarketModel(
```

Comment:
Why you don't use @Parcelize?

Reply:
Thanks for pointing that out. Since I didn't find the related dependency, I went with the older approach.
Additionally, the Parcelize library is designed specifically for Android. While it's possible to include this library in multiplatform projects, we should note that features might be significantly limited. When we fully migrate to KMP, we'll need to explore different approaches.